meilisearch/tests/search/errors.rs

```diff
@@ -37,104 +38,360 @@ async fn search_unexisting_parameter() {
 }
 
 #[actix_rt::test]
-async fn search_invalid_crop_marker() {
+async fn search_bad_q() {
     let server = Server::new().await;
     let index = server.index("test");
 
-    // object
-    let response = index.search_post(json!({"cropMarker":  { "marker": "<crop>" }})).await;
-    meili_snap::snapshot!(format!("{:#?}", response), @r###"
-    (
-        Object {
-            "message": String("invalid type: Map `{\"marker\":\"<crop>\"}`, expected a String at `.cropMarker`."),
-            "code": String("invalid_search_crop_marker"),
-            "type": String("invalid_request"),
-            "link": String("https://docs.meilisearch.com/errors#invalid-search-crop-marker"),
-        },
-        400,
-    )
+    let (response, code) = index.search_post(json!({"q": ["doggo"]})).await;
+    snapshot!(code, @"400 Bad Request");
+    snapshot!(json_string!(response), @r###"
+    {
+      "message": "invalid type: Sequence `[\"doggo\"]`, expected a String at `.q`.",
```

Comment:
We should have a better way to print `Sequence [`\"doggo\"`]` in the future

Reply:
Like what?
I would like to get rids of the term `Sequence` for json, but I think it’s a pretty good thing that we’re able to print `["doggo"]` is a json value